pkgs/development/libraries/science/chemistry/plumed/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, blas
+}:
+
+assert !blas.isILP64;
+
+stdenv.mkDerivation rec {
+  pname = "plumed";
+  version = "2.8.2";
+
+  src = fetchFromGitHub {
+    owner = "plumed";
+    repo = "plumed2";
+    rev = "v${version}";
+    hash = "sha256-ugYhJq8KFjT8rkAOX/yZ9IlEklXCwRxKH49REd2QN9E=";
+  };
+
+  postPatch = ''
+    patchShebangs .
```

Comment:
Is it possible to apply that only to one or more selected subdirectories?

Reply:
I guess it would be possible, yes. But why?

Reply:
To shorten the build time. `patchShebangs` is rather expensive shell code.

Reply:
Ah then probably no. I mean I could apply it to subdirectories but a lot of plumed toolings and compilation calls shell code. I could restrict it to subdirs but it would still be the majority of them so not shortening build time. Then again the C++ part takes quite some time to compile, by far outweighing the patching.